### DISCUSSION TITLE
suggestion of a new road Style

### DISCUSSION THREAD

**Discussion by BenaliSaidi** [liodali/osm_flutter#251] [Ideas]
first of all I have to thank you @liodali  for your great work 

then I just wanted to suggest a new style of road 
instead of drawing the road continuously 
so why not make it a dotted line like google map 

because the continuous line has some cracks

Thank you in advance

![Screenshot_20220416-040201](https://user-images.githubusercontent.com/48554787/163677423-5662e03d-b45b-4f20-922b-c992fa865b5a.png)

**Comment by liodali** [comment]
i will see if osmdroid and tangram es support this type of rendering

**Reply by BenaliSaidi** [reply]
Okk thanks
